packages/components/pie-form-label/src/form-label.scss

```diff
@@ -1 +1,33 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+.pie-form-label {
```

Comment:
I would name this `c-formLabel` so we can use the hyphens to denote hierarchy.

We should stick to using `.c-` format for consistency with other components (see pie-button, modal, toggle-switch). The use of `pie` is probably unnecessary too;